src/index.js

```diff
@@ -1,36 +1,108 @@
+/* eslint-disable no-unused-vars */
```

Comment:
Kindly ensure that you do not disable linter checks inline. They represent coding standards that you share with your team and most of the time, help to enforce best coding practices.